### DISCUSSION TITLE
how to solve "import error" when training

### DISCUSSION THREAD

**Discussion by realgump** [open-mmlab/mmocr#1808] [Q&A]
I am just install the environment by the step of readme. However, when I run:
```
python tools/train.py configs/textrecog/svtr/svtr-tiny_20e_st_mj.py
```

there's an error occurs:

```
ImportError: /Anaconda/envs/mmocr/lib/python3.8/site-packages/mmcv/_ext.cpython-38-x86_64-linux-gnu.so: undefined symbol: _ZN3c105ErrorC2ENS_14SourceLocationESs
```
I have reinstall the mmcv but it doesn't work.

**Comment by gaotongxiao** [comment]
MMCV has not been installed correctly, usually using `mim` to install `mmcv` again can prevent this from happening.

```bash
pip uninstall mmcv
pip install openmim
mim install 'mmcv>=2.0.0rc1'
```

If that doesn't work, you can paste the full installation log of MMCV here so that we can identify the problem.
